CONTRIBUTING

```diff
@@ -0,0 +1,121 @@
+# Contributing to Angular Codelab
+
+Thank you for deciding to contribute! We're excited to have you on the team 🙌
+
+Below you'll find our guidelines for contributing to further development of this codelab, which is hosted at [Angular NYC](https://github.com/angularNYC/angular-presentation) on GitHub.
+
+
+------
+
+
+#### Table Of Contents
+
+[Before You Start](#before-you-start)
+  * [Code of Conduct](#code-of-conduct)
+  * [NPM Packages](#npm-packages)
+
+[Contributing to the Codelab](#contributing-to-the-codelab)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Opening Issues on GitHub](#opening-issues-on-github)
+  * [Contributing Code](#contributing-code)
+  * [Making a Pull Request](#making-a-pull-request)
+
+[On Style](#on-style)
+  * [Git Commit Messages](#git-commit-messages)
+  * [Code Style](#code-style)
+
+
+-----
+
+
+## Before You Start
+
+
+### Code of Conduct
+
+This project adheres to the Contributor Covenant [code of conduct](CODE_OF_CONDUCT.md).
```

Comment:
Oops, bad link! Sorry 🙏

Do you mean link to one or paste it directly into the file? Is there one we prefer (if not this one)/should it be written from scratch?